src/index.js

```diff
@@ -16,6 +16,7 @@ const { errorHandler } = require("./middleware/error-handler");
 const { channelRouter } = require("./routes/channels/channels.router");
 const { pollsRouter } = require("./routes/polls/polls.router");
 const { resultRouter } = require("./routes/result/result.router");
+const { pollsRouter } = require("./routes/polls/polls.router");
```

Comment:
this looks like the same as line 17?

Reply:
I think I fixed everything now. Thanks for catching